_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,390 @@
+
+Copy the following sample configurations to set up and assemble the image into a Dockerfile.
+
+```json
+
+# Define function directory
+ARG FUNCTION_DIR="/function"
+# Base image of the docker container
+FROM node:lts-slim as build-image
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+# AWS Lambda runtime dependencies
+RUN apt-get update && \
+    apt-get install -y \
+        g++ \
+        make \
+        unzip \
+        libcurl4-openssl-dev \
+        autoconf \
+        automake \
+        libtool \
+        cmake \
+        python3 \
+        libkrb5-dev \
+        curl
+# Copy function code
+RUN mkdir -p ${FUNCTION_DIR}/
+COPY package.json src/ ${FUNCTION_DIR}/
+RUN ls ${FUNCTION_DIR}/
+WORKDIR ${FUNCTION_DIR}
+RUN npm install
+RUN npm install aws-lambda-ric
+# Build Stage 2: Copy Build Stage 1 files in to Stage 2. Install chromium dependencies and chromium.
+FROM node:lts-slim
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+# Set working directory to function root directory
+WORKDIR ${FUNCTION_DIR}
+# Copy in the build image dependencies
+COPY --from=build-image ${FUNCTION_DIR} ${FUNCTION_DIR}
+RUN ls ${FUNCTION_DIR}/
+# Install latest chrome dev package and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+# installs, work.
+RUN apt-get update \
+    && apt-get install -y wget gnupg \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 \
+      --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+ENTRYPOINT ["/usr/local/bin/npx", "aws-lambda-ric"]
+ENV IS_LAMBDA=true
+ENV HOME="/tmp"
+CMD [ "/function/index.handler" ]
+
+```
+
+### Step 2: Create a repository with Amazon ECR
+
+Follow the instructions in the Amazon ECR user guide to create a repository with the name `opensearch-reporting-cli`.
+
+Go to [Getting started with Amazon ECR using the AWS Management Console](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html) and follow both procedures.
+
+Navigate to the test function **Configuration** tab to modify two required settings:
+- Make sure to set **Timeout** to at least 5 minutes to allow the reporting CLI to generate the report.
+- Change the setting for **Ephemeral storage** to 1024MB. The default setting is not a sufficient storage amount.
+{: .note }
+
+### Step 3: Run the push commands
+
+In the AWS ECR console, choose **view push command**.
+Locate the 4 commands in the Dockerfile directory, and select them all.
```

Comment:
maybe something like Run all 4 commands sequentially in the same directory where above Dockerfile exist?
Users will run into errors if the Dockerfile is not in same directory